### PR TITLE
added Alan's project proposal

### PR DESCRIPTION
**Pull request by alanvww**

### CODE REVIEW

File: projects/README.md [hunk 1]
```diff
@@ -2,4 +2,5 @@
 
 Add your name, a link to your research proposal—see [template.md](template.md)—and a one sentence summary of your project.
 
-* name - [research project title](template.md) - one sentence description
+- name - [research project title](template.md) - one sentence description
+- Alan Ren - [Ml5.js Website Functionality Enhancement and Examples with Built-in Editor](alanren.md) - Enhance the ml5.js website and built-in code editor.
```
Comment (shiffman) [inline projects/README.md]:
```suggestion
- Alan Ren - [ml5.js Website Functionality Enhancement and Examples with Built-in Editor](alanren.md) - Enhance the ml5.js website and built-in code editor.
```

Tiny nitpick! ml5 and p5 generaly don't capitalize the first letter :)

**Review by shiffman — APPROVED** [review]
Looks great! Made one tiny "suggested" change if you want to practice accepting that before I merge!

**Comment by alanvww** [comment]
Just double check and I think this branch is ready to merge!

**Comment by shiffman** [comment]
🎉

**Pull request merged**